domains/authorization/migrations/SEED.sql

```diff
@@ -0,0 +1,69 @@
+DO $$
+DECLARE
+    organizations_empty boolean;
+    users_empty boolean;
+    affiliations_empty boolean;
+    terms_empty boolean;
+BEGIN
+
+SELECT COUNT(*) = 0 FROM "Terms" INTO terms_empty;
+IF terms_empty THEN
+        INSERT INTO "Terms" ("Id", "Version")
+        VALUES ('0ccb0348-3179-4b96-9be0-dc7ab1541771', 1);
+
+        RAISE NOTICE 'Terms table seeded successfully.';
+ELSE
+        RAISE NOTICE 'Terms table is not empty. Skipping seed data insertion.';
+END IF;
+
+SELECT COUNT(*) = 0 FROM "Organizations" INTO organizations_empty;
```

Comment:
Could we do an upsert instead so we make sure organizations are updated on a redeploy?